create_user.sh

```diff
@@ -2,13 +2,15 @@
 set -e
 
 MIRTE_SRC_DIR=/usr/local/src/mirte
-
 # Add mirte user with sudo rights
 #TODO: user without homedir (create homedir for user)
 sudo useradd -m -G sudo,audio -s /bin/bash mirte
 sudo mkdir /home/mirte/workdir
 sudo chown mirte:mirte /home/mirte/workdir
-echo -e "mirte_mirte\nmirte_mirte" | sudo passwd mirte
+
+sudo ./install_pam.sh # setup pam before changing the password for the mirte user.
+
+echo "mirte:{new_password}" | sudo chpasswd
```

Comment:
Where is {new_password} set?

Reply:
nowhere, probably some find replace mistake